cmd/integration/commands/stages.go

```diff
@@ -252,6 +264,15 @@ func stageIHash(ctx context.Context) error {
 	db := ethdb.MustOpen(chaindata)
 	defer db.Close()
 
+	if err := migrations.NewMigrator().Apply(db, ""); err != nil {
```

Comment:
Do you think the second argument needs to be `datadir`? And in other places

Reply:
fixed